tests/orquestra/quantum/estimation/estimation_test.py

```diff
@@ -17,18 +17,18 @@
     split_estimation_tasks_to_measure,
 )
 from orquestra.quantum.measurements import ExpectationValues, Measurements
-from orquestra.quantum.openfermion import IsingOperator, QubitOperator
 from orquestra.quantum.symbolic_simulator import SymbolicSimulator
 from orquestra.quantum.testing import MockQuantumBackend
+from orquestra.quantum.wip.operators import PauliSum, PauliTerm
 
 
 class TestEstimatorUtils:
     @pytest.fixture()
     def frame_operators(self):
         operators = [
-            IsingOperator("2 [Z1 Z2]"),
-            IsingOperator("[Z0 Z3]"),
-            IsingOperator("-1 [Z2]"),
+            PauliTerm.from_str("2*Z1*Z2"),
```

Comment:
Hmm...
Should we keep an option to create those PauliTerms/Sums from openfermion strings? Just for compatibility reasons 🤔 ?
CC: @dexter2206

Reply:
Also will do in separate PR